core/controllers/suggestion.py

```diff
@@ -182,6 +191,7 @@ def put(self, target_id, suggestion_id):
                 # The skill_id is passed only at the time of accepting the
                 # suggestion.
                 suggestion.change.skill_id = target_id
+            suggestion.skill_difficulty = self.payload.get('skill_difficulty')
```

Comment:
As per our discussion, skill_difficulty is not going to be the part of suggestion object right?

Reply:
We had thought of handling this in a way we handle commitMessage in exploration translation/content suggestion. See [here](https://github.com/oppia/oppia/blob/ed4f3ba5a0714c5c09cc2cee4208ad6171edd0a5/core/domain/suggestion_registry.py#L343) commit_message isn't the part of the suggestion model*

Reply:
I'm not sure how we could fetch the difficulty from somewhere else during the accept since the difficulty is chosen upon review submission and so must be tied to the submitted suggestion somehow. The skill difficulty was only added to the SuggestionAddQuestion not to the BaseSuggestion class, so it shouldn't affect other types of suggestions.

Reply:
`SuggestionAddQuestion` is stored in the GeneralSuggestionModel which doesn't have this new property, right?

I cannot think of a way to do this, I was thinking to add commit_message and skill_difficulty as default param for `suggestion_services.accept_suggestion` but that doesn't sounds like a good idea as the `suggestion_services.accept_suggestion` is a general function. Let me know your thoughts on how we can/should handle this?